IssueComment
  { issueCommentUpdatedAt = 2016 (-06) (-09) 21 : 34 : 08 UTC
  , issueCommentUser =
      SimpleUser
        { simpleUserId = Id 505365
        , simpleUserLogin = N "danielmewes"
        , simpleUserAvatarUrl =
            "https://avatars.githubusercontent.com/u/505365?v=3"
        , simpleUserUrl = "https://api.github.com/users/danielmewes"
        , simpleUserType = OwnerUser
        }
  , issueCommentUrl =
      "https://api.github.com/repos/rethinkdb/rethinkdb/issues/comments/225033717"
  , issueCommentHtmlUrl =
      "https://github.com/rethinkdb/rethinkdb/issues/5840#issuecomment-225033717"
  , issueCommentCreatedAt = 2016 (-06) (-09) 21 : 34 : 08 UTC
  , issueCommentBody =
      "Hmm actually does `emergency_repair: unsafe_rollback` not work when you start the non-voting replica in an isolated (empty) cluster? If that is the case, I think that's a bug or oversight on our end, and we need to look into that."
  , issueCommentId = 225033717
  }